musicbot/bot.py

```diff
@@ -3129,6 +3132,41 @@ def _get_url() -> str:
                 )
             return url
 
+        if option in ["+all", "add_all"]:
+            if not player.playlist.entries:
+                raise exceptions.CommandError(
+                    self.str.get(
+                        "cmd-autoplaylist-add-all-empty-queue",
+                        "The queue is empty. Add some songs with `{0}play`!",
+                    ).format(self.server_data[guild.id].command_prefix),
+                    expire_in=30,
+                )
+
+            added_songs = set()
+            for e in player.playlist.entries:
+                url = e.url
+                self._do_song_blocklist_check(url)
+                if e.url not in self.server_data[guild.id].autoplaylist:
+                    await self.server_data[guild.id].autoplaylist.add_track(url)
+                    added_songs.add(url)
```

Comment:
Just a nit-pick more than anything, but why not use `e.url` in place of all the `url` in this loop here?

Reply:
copy paste :p